pyproject.toml

```diff
@@ -59,3 +59,10 @@ exclude = '''/(
 [tool.isort]
 profile = "black"
 known_first_party = ["llm_app"]
+
+[tool.mypy]
+python_version = "3.10"
+exclude = ['examples']
+ignore_missing_imports = "True"
+follow_imports_for_stubs = "True"
+check_untyped_defs = "True"
```

Comment:
@embe-pw WDYT?

Reply:
```suggestion
[tool.mypy]
python_version = "3.10"
ignore_missing_imports = true
follow_imports_for_stubs = true
check_untyped_defs = true
```

Reply:
(also, could you please check if we can disable `ignore_missing_imports = true`?)

Reply:
It seems to me that we should stay with `ignore_missing_imports = true` and `exclude 'examples'`at least for now.

Reply:
ok, but change the bools from strings to bools anyway :)